go/src/koding/klientctl/endpoint/auth/auth.go

```diff
@@ -44,10 +44,11 @@ type LoginOptions struct {
 	Token    string
 	Username string
 	Password string
+	Prefix   string
 }
 
 func (opts *LoginOptions) AskUserPass() (err error) {
-	opts.Username, err = helper.Ask("Username [%s]: ", config.CurrentUser.Username)
+	opts.Username, err = helper.Ask("%sUsername [%s]: ", opts.Prefix, config.CurrentUser.Username)
```

Comment:
Just out of curiosity, how `opts.Prefix` can look like? Will it be `prefixUsername [user]` or `PrefixUsername [user]` ?